packages.bzl

```diff
@@ -1,7 +1,7 @@
 # Each individual package uses a placeholder for the version of Angular to ensure they're
 # all in-sync. This map is passed to each ng_package rule to stamp out the appropriate
 # version for the placeholders.
-ANGULAR_PACKAGE_VERSION = "^15.0.0 || ^16.0.0"
+ANGULAR_PACKAGE_VERSION = "^15.0.0-0 || ^15.1.0-0 || ^15.2.0-0 || ^15.3.0-0 || ^16.0.0-0"
```

Comment:
Should the 15.x ones be tildes (`~`)?   so that its everything in the minor range?

Reply:
tilde only allows for all patch versions to my knowledge.

Reply:
Correct, I guess what I mean is that `^15.0.0-0 || ^15.1.0-0 || ^15.2.0-0 || ^15.3.0-0` is redundant, whereas `~15.0.0-0 || ~15.1.0-0 || ~15.2.0-0 || ~15.3.0-0` would be all of the patch versions in each minor.

I might be misunderstanding what it does, or what your goal is here.

Reply:
If you just read it, then yes, those seem redundant- but that's just how NPM works. See the comment. NPM **requires** every minor to be explicitly specified. What you proposed would also work, but having at least one `^` range ensures that even unlisted non-prerelease minors are supported. e.g. `15.4.0`

Reply:
Okay, I definitely don't want to block on it and I can read up more to understand it.

Reply:
I can give a few more examples. No worries, sorry if I phrased it confusingly:

e.g. `^15.0.0-0` allows for: `15.0.0`, or `15.1.0` but not for `15.1.0-next.0`.
e.g. `~15.0.0-0 || ~15.1.0-0` allows for: `15.0.0`, or `15.0.0-next.0`, or `15.1.0-next.0`- but not for `15.2.0`.
  &nbsp; &nbsp; in comparison: `^15.0.0-0 || ~15.1.0-0` would allow for all above

Reply:
Unfortunately not in NPM. See https://classic.yarnpkg.com/en/docs/dependency-versions#toc-pre-release-tags